packages/dds/tree/src/shared-tree-core/editManager.ts

```diff
@@ -318,6 +336,13 @@ export class EditManager<
 			const newTrunkBase = latestEvicted as Mutable<typeof latestEvicted>;
 			// The metadata for new trunk base revision needs to be deleted before modifying it.
 			this.trunkMetadata.delete(newTrunkBase.revision);
+			// collect the revisions that will be trimmed to send as part of the branch trimmed event
+			const trimmedRevisions: RevisionTag[] = [];
+			let currCommit: GraphCommit<TChangeset> | undefined = newTrunkBase;
```

Comment:
I'd recommend just writing this as `getPathFromBase(newTrunkBase, this.trunkBase).map((c) => c.revision)`. While it may technically be less efficient (two loops instead of one), it keeps all of our "parent walking" code centralized. That way, when we eventually do go to optimize EditManager, it will hopefully be easier to analyze.